examples/recording_service/service_admin/cpp/Requester.cxx

```diff
@@ -49,59 +49,67 @@ uint32_t ArgumentsParser::parse_domain_id(char *arg)
     stream >> domain_id;
     if (stream.bad()) {
         std::stringstream error_stream;
-        error_stream << "Error: could not parse uint32 value provided, '"
-                << arg << "'";
+        error_stream << "Error: could not parse uint32 value provided, '" << arg
+                     << "'";
```

Comment:
This looks weird. I would have expected this to be indented as the parameter for a function.